test/runtests.jl

```diff
@@ -43,6 +43,7 @@ T = Float32
         @test isapprox( signalsum, T(2), atol = 5e-3 )
         nt = NamedTuple(sim)
         @test sim == Simulation(nt)
+        #@test isapprox(get_depletion_voltage(sim, 1, 1:5000, verbose = false), T(3053), atol = 2.0) 
```

Comment:
There is an effect especially prominent for 2D geometries, which results in varying depletion voltages, depending on the refinement parameters from simulate!. When choosing smaller convergence limits or refinement limits, there is fluctuation in the electric potential, which affects the local min max search. This test for example yields 3053V for default `simulate!` parameters, and 440533V for the parameters chosen above.
How should we deal with that? Do we expect the user to know about this effect or should we adapt the algorithm, in order to be less sensitive to noise?

Reply:
These values seems way to high to me... I would expect that the depletion voltage would be of the order of 2000V and not fluctuate that much. 

What I think happens here is that the detector is not biased in reversed bias mode.
In the config file, we apply 0V to the core (id: 1) and 3500V to the mantle (id: 2) contact. 
What happens in this algorithm is that we set all contacts to 0V and then vary the potential that we apply to the core contact (id: 1). In this scenario, we would need to apply a **NEGATIVE** voltage to the core. 

Can you try if you choose the range from -1:-1:-5000 if the result is still unstable?

Reply:
Otherwise, you would need to call
`get_depletion_voltage(sim, 2, 1:5000)` (for the mantle which has id: 2)

Reply:
A good test would also be to check if the result is somewhat reasonable.

You can save the output of the depletion voltage and simulate the detector 10V above and below the depletion voltage with `depletion_handling = true` and check whether it matches the expected depletion behavior.

Something like
```julia
# Determine depletion voltage
id = 2
deplV = get_depletion_voltage(sim, contact_id)

# Simulate the detector 10V below depletion voltage => should be undepleted
sim.detector = SolidStateDetector(sim.detector, contact_id = id, contact_potential = deplV - 10)
calculate_electric_potential!(sim, depletion_handling = true)
@test !is_depleted(sim.point_types)

# Simulate the detector 10V above the depletion voltage => should be depleted
sim.detector = SolidStateDetector(sim.detector, contact_id = id, contact_potential = deplV + 10)
calculate_electric_potential!(sim, depletion_handling = true)
@test is_depleted(sim.point_types)
```
```

Reply:
So, for the inverted coaxial detector, this should be quite a good approximation:
```julia
sim = Simulation(SSD_examples[:InvertedCoax])
simulate!(sim, depletion_handling = true, refinement_limits = [0.2,0.1,0.05,0.03,0.01])
U = get_depletion_voltage(sim, 2)
```
gives a depletion voltage of `U = 1855.0f0`.

If we use this depletion voltage for the simulation, i.e.
```julia
sim.detector = SolidStateDetector(sim.detector, contact_id = 2, contact_potential = U)
calculate_electric_potential!(sim, depletion_handling = true, refinement_limits = [0.2,0.1,0.05,0.03,0.01])
plot(sim.point_types, full_det = true)
```
![image](https://user-images.githubusercontent.com/30291312/183932646-f96cc244-9ed6-4356-884f-7211e31b01f1.png)

whereas if we simulate the detector with 20V more voltage, we fully deplete it:
```julia
sim.detector = SolidStateDetector(sim.detector, contact_id = 2, contact_potential = U + 20)
calculate_electric_potential!(sim, depletion_handling = true, refinement_limits = [0.2,0.1,0.05,0.03,0.01])
plot(sim.point_types, full_det = true)
```
![image](https://user-images.githubusercontent.com/30291312/183933159-57133ca2-c5c6-4d9e-85d1-aba3b1377691.png)

Reply:
You were right, for a negative voltage at contact 1, the depletion voltage is more reasonable. Still there is a rather big difference in the depletion voltage between the (more refined) default refinement parameters and the parameters used during testing. Default parameters yield -1827V and test parameters -1923V

Reply:
I guess the difference also depend on the refinement limits.
The default, I believe, stops at a refinement limit of 5%, whereas the test have a refinement limit of 10%.
Therefore, I would also assume a 5% uncertainty `(-1827 ± 91)V` on the depletion voltage for the default parameters and a 10% uncertainty `(-1923 ± 192)V` on the test parameters.